subscription.go

```diff
@@ -193,7 +192,15 @@ func (r *marathonClient) registerSSESubscription() error {
 	return nil
 }
 
-// Unsubscribe removes ourselves from Marathon's callback facility
+// Subscribe adds a URL to Marathon's callback facility
+//	url		: the url you wish to subscribe
```

Comment:
Name of the variable is actually `callbackURL`.

Should also say `URL` in the description.